build/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM quay.io/operator-framework/helm-operator:v0.7.0
+
```

Comment:
Does the operator-framework enforce any further DOCKER args etc. Can be taken up as part of the submission.

Reply:
Not sure @kmova ..will explore on this/update here based on submission feedback.